elements/pl-matching/pl-matching.css

```diff
@@ -0,0 +1,35 @@
+
+.matching-statements > div {
+    display: flex;
+    align-items: baseline;
+    margin-top: 1ex;
+}
+
+.matching-statements > div > div {
+    margin-right: 1ex;
+}
```

Comment:
These seem like pretty generic, and I'm afraid a future change may inadvertently add a div that triggers (or not) this formatting in a way that is unintended. I'd be more in favour of adding classes to these divs, and using those in the CSS blocks. Also, nitpick, I think it may be better to have a `pl-` prefix (e.g., `pl-matching-statements` to avoid potential conflict with other elements outside the `pl-matching` element itself.